app/Filament/Resources/EnquiryResource.php

```diff
@@ -133,56 +136,94 @@ public static function infolist(Infolist $infolist): Infolist
+                                            ->placeholder('N/A'),
+                                        TextEntry::make('city')
+                                            ->label('City')
+                                            ->placeholder('N/A'),
+                                        TextEntry::make('pincode')->label('PIN Code'),
+                                    ])
+                                    ->columns(4),
+                            ]),
                         Section::make('Preferences')
+                            ->columnSpan([
+                                'default' => 4,
+                                'sm'      => 1,
+                                'xl'      => 2,
+                            ])
+                            ->columns([
+                                'sm' => 1,
+                                'md' => 2,
+                            ])
                             ->schema([
                                 TextEntry::make('interested_in')
                                     ->label('Interested In')
-                                    ->hidden(fn($record) => empty($record->interested_in)),
+                                    ->columnSpanFull()
+                                    ->placeholder('N/A'),
                                 TextEntry::make('source')
                                     ->label('Source'),
-                                TextEntry::make('why_do_you_plan_to_join')
-                                    ->label('Reason for Joining'),
-                                TextEntry::make('start_by')
-                                    ->label('Preferred Start Date')
-                                    ->date('d-m-Y')
-                                    ->hidden(fn($record) => empty($record->start_by)),
-                            ])
-                            ->columns(2)
-                            ->columnSpan(2),
-                    ]),
-                Section::make('Address')
-                    ->schema([
-                        TextEntry::make('address')->label('Address'),
-                        Group::make()
-                            ->schema([
-                                TextEntry::make('country')->label('Country'),
-                                TextEntry::make('state')
-                                    ->label('State')
-                                    ->hidden(fn($record) => empty($record->state)),
-                                TextEntry::make('city')
-                                    ->label('City')
-                                    ->hidden(fn($record) => empty($record->city)),
-                                TextEntry::make('pincode')->label('PIN Code'),
-                            ])
-                            ->columns(4),
-                    ]),
-            ]);
+                                TextEntry::make('fitness_goal')
+                                    ->label('Fitness goal ?'),
```

Comment:
```suggestion
                                TextEntry::make('goal')
                                    ->label('Goal ?'),
```

Would be good it keep it generic

Reply:
Changes Done ✅